docs/ucan-invocation-stream.md

```diff
@@ -53,3 +53,33 @@ Consumers might need other infrastructure resources to track state based on the
 The `admin-metrics` table has a partition key `name` with the metric name we keep track. With this, we can easily update and query each of the `admin` metrics we care about.
 
 In the context of `space-metrics` table, a partition key with `space` is used together with a sort key `name` with the metric name. This way, we are able to track and query each metric for a given space.
+
+### Delivery Firehose
+
+At the end of the invocation stream we archive UCAN invocations and receipts as JSON
+logs stored in S3. We partition log storage by "type" (ie, `workflow` or `receipt`),
+"op" (ie, the UCAN's ability - `store/add`, `upload/remove`, etc) and "day"
+(a `%Y-%m-%d` formatted string). We designed partitioning this way to make it easy to 
+find logs for a particular operation on a particular day, which is used extensively
+by AWS Athena to make UCAN log queries efficient.
+
+For example, receipts of the `store/add` operation from January 1, 2024 are stored in:
+
+`/logs/receipt/store/add/2024-01-01`
+
+#### Glue
```

Comment:
could this link to where glue-related code is in w3infra? (if any). glue is new to me

Reply:
ah great idea, done!